app/controllers/api/v1/accounts/lookup_controller.rb

```diff
@@ -1,10 +1,14 @@
 # frozen_string_literal: true
 
 class Api::V1::Accounts::LookupController < Api::BaseController
+  skip_before_action :require_authenticated_user!
```

Comment:
You use the `only` param in the other controllers and I think it's a good idea to do the same here. Yeah, you might skip this action for all controller methods, but at least you're being explicit about it. And you're less likely to get bitten by someone adding a new method that should check for authenticated users.